### ISSUE TITLE
Mark of Penitence spell suggestion

### ISSUE THREAD

**Issue by Marcloure** [mlenser/kryx-rpg-issues#838]
#### Holy | 5 mana

As you spend 10 minutes to cast this spell, you brand a willing or restrained creature within 1 meter of you with a visible mark. The creature must remain within 1 meter of you for the time it takes to cast this spell. Once the creature is marked, designate an action or a kind of behavior that the mark forbids, and select one of the following penitences or another worked out by you and your GM. When the target performs the forbidden action or displays the forbidden behavior, it is affected by the penitence until a cure is found (usually another 5 mana spell). Example penitences include the following:

***Penitence of Exile.*** The creature is regarded with hostility by other creatures, from animals to loved ones. A remove curse spell augmented to 5 mana or a sympathy spell augmented to 5 mana cast on the creature removes this penitence.

***Penitence of Fatigue.*** The creature is exhausted 5. It can't reduce its exhaustion level below 5 until a restoration spell augmented to 5 mana is cast on it.

***Penitence of Oblivion.*** The creature forgets the majority of its life, including any person it knows or interests it has. A remove curse spell augmented to 5 mana or restoration spell augmented to 5 mana cast on the target restores the creature’s memory.

***Penitence of Solitude.*** The creature's eyes burst in flames and it bleeds from its ears. The creature becomes blinded and deafened. Only a restoration spell augmented to 5 mana or a wish spell can restore the creature's senses.

No matter the distance, you are aware when the subject triggers the penitence or when the penitence is lifted.

A remove curse spell augmented to 5 mana or a wish spell cast on the target removes the Mark of Penitence without triggering its effect.

---

This spell is a way for priests to give their enemies a second chance to repent. Yes, it does have some pretty horrible effects.

I based it on the [Mark of Justice](https://dnd4.fandom.com/wiki/Mark_of_Justice) ritual from 4E. It is quite strong and has no save, but it requires the target to be incapacitated for 10 minutes to be performed. You could just kill the target in that time if that was the goal. It also has no effect until the specified action or behaviour is triggered.

**Comment by mlenser** [comment]
Why not just disallow the specified action instead of having a penalty if they perform the specified action?

**Comment by DalenWBrauner** [comment]
Because that's not quite the point of the spell :) I think it creates some pretty awesome roleplay opportunities- the secret backstory of an amnesiac, the oppressive demands of a local cult or church, a popular leader blackmailed into exile...

Besides, a creature affected by the curse can choose to repeat the offending behavior as much as they like once afflicted; they're never truly prevented from the action in the first place. That choice is theirs.

**Comment by mlenser** [comment]
Philosophically: It's not really free will if you punish the outcome you don't want. Effectively it's not much different from preventing the action entirely. One could argue it's more sadistic to even allow it in the first place.

Mechanically: This is worse than Geas (a 3 mana spell). It needs to be scaled to lower mana and have the mechanics updated based on Gaes.

Also, targetting only willing or restrained creatures is very arbitrary and doesn't even evoke the thematics I assume the goal was.  If the goal is what I expect, then there are several other conditions like Paralyzed and Petrified that should also qualify. But I think that's the wrong direction entirely. I would change it to any creature like Geas is.

**Comment by itamarcu** [comment]
I like the idea, and it was used pretty well in Order of the Stick ([Greater Mark of Justice](https://oots.fandom.com/wiki/Greater_Mark_of_Justice)).  But, I'm not sure if it needs to be an actual spell in the system.  If a player needs to be affected the DM is allowed to make up magical oaths/curses/restraints for the special scenario, and if a very high level player wants to cast this on an NPC, it's not as interesting and not too different than Geas.

Still, it would be nice to have a spell that's weaker than Geas (which makes you charmed and forces you to obey every instruction).  Something where you forbid a single thing and the duration is permanent, and then it can be 3-4 mana with a smaller punishment.

for "willing or restrained creature" - if the spell already takes 10 minutes, those requirements are unnecessary.

**Comment by Marcloure** [comment]
> Targetting only willing or restrained creatures

You need to brand the creature with a physical mark, like a stigmata or the symbol of your deity maybe. So it shouldn't be able to move around.

> This is worse than Geas (a 3 mana spell).

To last indefinitely, Geas needs to be cast at 5 mana. Also, Geas is different in that the creature can continue to act against your command if it doesn't care to much about the damage (maybe it can heal, or it's only a short act that 5d10 won't matter). Also, Geas works better when you don't want the creature to drastically change, just to punish it's misconduct.

Mark of Penitence is intended to give a creature a second chance for them to go on about their lives, but if they continue their wrongdoings, they are stripped of all they had. It's similar to Gears in some ways, but I think it's much harsher than constantly take damage.

> Effectively it's not much different from preventing the action entirely.

The creature can still do the act, but only once.

---

Sincerely, I'm fine if it is just a version of Geas in holy too. I was just thinking that holy is missing a spell to force sinners and wrongdoers to repent, or at least to pretend they repented. Sure, Fear is in the Divine power source, so maybe that is enough...

**Comment by mlenser** [comment]
> holy is missing a spell to force sinners and wrongdoers to repent

"Force them to repent"... That sounds quite like a retribution theme more than Holy. I don't think it belongs in Holy as it currently exists nor do I think Holy should encorporate retribution type stuff 

_____________

I don't see a place for this spell as it currently exists. If it is refined to be something similar to Geas that augments to something strong at 5 mana then it'd fit better, but the theme problem would still exist. And I think thematically "you can do this once, but then are punished" is really not a very good design. Either prevent it entirely or have meaningful punishments that don't completely debilitate the creature.

If this is refined it can be reconsidered.
